frontends/concrete-python/examples/prime-match/prime-match-sh.py

```diff
@@ -0,0 +1,113 @@
+import random
```

Comment:
I would rename prime-match-sh.py to prime-match-semi-honest.py. I read 'sh' as 'bash' the first time

Reply:
Agreed, another alternative is `semi-honest-prime-match.py`.

Reply:
done